guide/16-geoenrichment-guides/part5_generate_reports.ipynb

```diff
@@ -0,0 +1,928 @@
+{
```

Comment:
<ol><li>The area_options should be listed outside of the customizing reports, since readers tend to think customization usu. handles how the output or format would be different from default. I would recommend putting the "Using Options" section to where bufferedReport example ends. So we can just provide it as an alternative...</li><li>"Reports can be customized by specifying optional paramaters. Report header can be customzied to include title, subtitle and logo etc. " can be changed to:</li><li class="ql-indent-1">"The optional parameter <code>report_fields</code> specifies additional choices to customize reports. For example, the title, subtitle, logo, etc., which appear in the header of a report can be customized with this parameter."</li></ol>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/755/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>